it/grpc/java/build.gradle

```diff
@@ -1,4 +1,5 @@
 dependencies {
     implementation project(':grpc')
     implementation libs.grpc.netty.shaded
+    testImplementation libs.javax.annotation
```

Comment:
The generated gRPC stubs need `@javax.annotation.Generated`. I have no idea why it worked before. 🤔